src/components/CreateCoursePage/CreateCourseForm.jsx

```diff
@@ -36,15 +33,17 @@ class CreateCourseForm extends React.Component {
       organizations,
       pristine,
       submitting,
+      isCreating,
```

Comment:
What's the difference between submitting and isCreating?

Reply:
Submitting is part of the redux-forms properties.  It switches to false before the creation actions are complete.  I spent some time trying to get it to wait for the API calls but decided to use this flag instead when I was unable to identify the issue.